VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -95,10 +96,8 @@ public void onCreate(Bundle savedInstanceState) {
             getAccountTask = new GetAccountTask(getActivity(), profile.username) {
                 @Override
                 protected void onSuccess(@NonNull Account account) throws Exception {
-                    NavigationFragment.this.profileImage = account.getProfileImage();
-                    if (null != imageView) {
-                        loadProfileImage(account.getProfileImage(), imageView);
-                    }
+                    profileImage = account.getProfileImage();
```

Comment:
This logic is actually useful for handling the case where the view is (re)created _after_ the image has been loaded. It doesn't seem to happen in the current implementation, but that might change, and the Fragment should be prepared to handle this case.